front/src/modules/rollingStock/components/RollingStockSelector/RollingStockCurve.tsx

```diff
@@ -106,17 +119,37 @@ function Legend(props: {
           tabIndex={0}
           onClick={() => changeCurveState(curve.id)}
         >
-          {curve.mode}
-          {curve.comfort !== 'STANDARD' ? comfort2pictogram(curve.comfort) : null}
+          {isOnEditionMode && (
+            <>{showPowerRestriction ? curve.power_restriction : curve.electrical_profile_level}</>
+          )}
+          {!isOnEditionMode && !showPowerRestriction && curve.mode}
+          {curve.comfort !== 'STANDARD' && !isOnEditionMode
```

Comment:
no inline string for 'STANDART', use a constant identifier

Reply:
Done :)